simgear-boost-1.73.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -227,7 +227,7 @@
+ endif()
+ 
+ find_package(Boost REQUIRED)
+-set (BOOST_CXX_FLAGS "-DBOOST_BIMAP_DISABLE_SERIALIZATION")
++set (BOOST_CXX_FLAGS "-DBOOST_BIMAP_DISABLE_SERIALIZATION -DBOOST_NO_STDLIB_CONFIG")
```

Comment:
well, this patch isn't required anymore, I checked in the simgear repo and there is already this line in it.

Reply:
I guess we need to update to 2020.1.3 first.

Reply:
Oh if it's for the next version you can either keep them or update to 2020.1.3, at your convenience.